homeassistant/components/sonos/speaker.py

```diff
@@ -277,19 +264,11 @@ def setup(self, entry: ConfigEntry) -> None:
         dispatcher_send(self.hass, SONOS_CREATE_MEDIA_PLAYER, self)
         dispatcher_send(self.hass, SONOS_SPEAKER_ADDED, self.soco.uid)
 
+        self.hass.add_job(self.async_subscribe())
```

Comment:
```suggestion
        self.hass.create_task(self.async_subscribe())
```

This should work an give proper typing should sonos get to a point where it can be strictly typed